anomalous-token-transfers-ml-py/README.md

```diff
@@ -0,0 +1,182 @@
+    'LOOKS_value',
+    'MANA_transfers',
+    'MANA_value',
+    'MATIC_transfers',
+    'MATIC_value',
+    'SAITAMA_transfers',
+    'SAITAMA_value',
+    'SAND_transfers',
+    'SAND_value',
+    'SHIB_transfers',
+    'SHIB_value',
+    'SOS_transfers',
+    'SOS_value',
+    'STRNGR_transfers',
+    'STRNGR_value',
+    'STRONG_transfers',
+    'STRONG_value',
+    'USDC_transfers',
+    'USDC_value',
+    'USDT_transfers',
+    'USDT_value',
+    'WBTC_transfers',
+    'WBTC_value',
+    'WETH_transfers',
+    'WETH_value',
+    'account_age_in_minutes', # diff between tx from address's first and tx that's being evaluated
+    'max_single_token_transfers', # max token transfer count and value. Could be a token that's not in the top 20.
+    'max_single_token_transfers_value',
+    'tokens_type_counts', # unique number of token types transferred
+    'transfer_counts' # total number of token transfer events
+]
+```
+
+## Supported Chains
+
+- Ethereum
+
+## Alerts
+
+- NORMAL-TOKEN-TRANSFERS-TX
+  - Fired when a transaction is predicted as normal/inlier.
+  - Severity is always set to "info"
+  - Type is always set to "info"
+  - Metadata will include features and model prediction values. It will also include feature generation and prediction response times.
+- ANOMALOUS-TOKEN-TRANSFERS-TX
+  - Fired when a transaction is predicted as anomalous/outlier.
+  - Severity is always set to "low" (mention any conditions where it could be something else)
+  - Type is always set to "info" (mention any conditions where it could be something else)
+  - Metadata will include features and model prediction values. It will also include feature generation and prediction response times.
+- INVALID-TOKEN-TRANSFERS-TX
+  - Fired when bot fails to generate valid model features. Model will not make predictions on invalid features.
+  - Severity is always set to "low"
+  - Type is always set to "info"
+  - Metadata will not include a model prediction.
+
+## Test Data
+
+### Normal Tx Example
+
+```bash
+$ npm run tx 0x404666af36d5f2e11f763391be0a5b40ae78dfd4304b4f22e3a53c369e779bf1
+
+1 findings for transaction 0x404666af36d5f2e11f763391be0a5b40ae78dfd4304b4f22e3a53c369e779bf1 {
+  "name": "Normal Transaction",
+  "description": "0x1b4006139cfd127822ff9b26a0f89532977b3a42 executed normal tx with token transfers",
+  "alertId": "NORMAL-TOKEN-TRANSFERS-TX",
+  "protocol": "ethereum",
+  "severity": "Info",
+  "type": "Info",
+  "metadata": {
+    "from": "0x1b4006139cfd127822ff9b26a0f89532977b3a42",
+    "transfer_counts": 1,
+    "account_age_in_minutes": 0,
+    "USDT_transfers": 1,
+    "USDT_value": 1700,
+    "token_types": [
+      "Tether USD-USDT"
+    ],
+    "max_single_token_transfers_name": "Tether USD",
+    "tokens_type_counts": 1,
+    "max_single_token_transfers": 1,
+    "max_single_token_transfers_value": 1700,
+    "feature_generation_response_time": 3.8916723749999997,
+    "model_prediction": "NORMAL",
+    "model_score": 0.189,
```

Comment:
there you have the score. great. Can we normalize this to 0-1? I think we want to establish a standard that gets followed by other bot deveopers.